src/geo/leaflet/leaflet-layer-view-factory.js

```diff
@@ -0,0 +1,49 @@
+var log = require('cdb.log');
```

Comment:
The introduction of this factory is arguably better what we had before so :+1: for me under current circumstances. Anyway, since you asked for thoughts on this particularly: 

1. The `LeafletTorqueLayer` is actually one of those cases that IMHO could/should be _registered_ dynamically when that bridge-lib loads, instead of having this factory even have to know anything about it.

2. It's assumed all layers here are created in the same way. That seems fine for the current use-cases, but will not work the second there's a new type that can't adhere to this convention, which is why I prefer the approach of delegating the object creation entirely.